pallets/registrar/src/lib.rs

```diff
@@ -229,6 +229,10 @@ pub mod pallet {
     #[pallet::getter(fn registrar_deposit)]
     pub type RegistrarDeposit<T: Config> = StorageMap<_, Blake2_128Concat, ParaId, DepositInfo<T>>;
 
+    #[pallet::storage]
+    pub type ParaManager<T: Config> =
+        StorageMap<_, Blake2_128Concat, ParaId, T::AccountId, OptionQuery>;
```

Comment:
`ParaManager` should be removed on deregister, in `cleanup_deregistered_para_id`